src/Nethermind/Nethermind.Init/InitializeStateDb.cs

```diff
@@ -104,11 +104,15 @@ public Task Execute(CancellationToken cancellationToken)
                 if (_logger.IsWarn) _logger.Warn($"Detected {pruningConfig.CacheMb}MB of pruning cache config. Pruning cache more than 2000MB is not recommended as it may cause long memory pruning time which affect attestation.");
             }
 
+            // 8 bytes Node pointer, (LinkedListNode 8 pointer + 8 pointer) + (HashAndTinyPath: 32 bytes for value + 8 bytes for path) + 32 bytes ValueHash256
+            int KeyEntryMemorySize = 8 + (8 + 8) + (32 + 8) + 32; // 96 bytes
```

Comment:
Can you double the default config also.

Reply:
Does it need 2.1M hashes?

Reply:
GC time is a function of number of objects (rather than size) I can redesign an Lru so it uses a giant array and indexes rather than pointers, if you want to go this high

Reply:
Yes, I've specifically, tune this amount.

Reply:
PR to eradicate the objects https://github.com/NethermindEth/nethermind/pull/7100